src/CMakeLists.txt

```diff
@@ -50,9 +47,31 @@ if (“${PROJECT_SOURCE_DIR}” STREQUAL “${CMAKE_SOURCE_DIR}”)
         set(BLT_CXX_STD "c++11" CACHE STRING "")
     endif()
 
-    set(ENABLE_ASTYLE      OFF CACHE BOOL "")
-    set(ENABLE_CLANGTIDY   OFF CACHE BOOL "")
-    set(ENABLE_UNCRUSTIFY  OFF CACHE BOOL "")
+    # These are not used in Axom, turn them off
+    set(AXOM_NOT_USED_BLT_TOOLS
+        CLANGQUERY
+        CLANGTIDY
+        VALGRIND
+        ASTYLE
+        CMAKEFORMAT
+        UNCRUSTIFY
+        YAPF)
+    foreach(_tool ${AXOM_NOT_USED_BLT_TOOLS})
+        set(ENABLE_${_tool} OFF CACHE BOOL "")
+    endforeach()
+
+    # These are only used by Axom developers, so turn them off
+    # unless an explicit executable path is given
+    set(AXOM_USED_BLT_TOOLS
+        CLANGFORMAT
+        CPPCHECK
+        DOXYGEN
+        SPHINX)
+    foreach(_tool ${AXOM_NOT_USED_BLT_TOOLS})
+        if(NOT ${_tool}_EXECUTABLE)
+            set(ENABLE_${_tool} OFF CACHE BOOL "")
+        endif()
```

Comment:
Do we need the other leg of the `if`?  So, if the user has defined `DOXYGEN_EXECUTABLE`, do we want to have `set(ENABLE_${_tool} ON CACHE BOOL "")` or is that not needed?

Reply:
Don't need to but will guard against when i possibly turn these off by default in BLT.